src/modules/general/components/multiSelect/multiSelect.tsx

```diff
@@ -38,7 +38,10 @@ const MultiSelect: React.FC<MultiSelectProps> = props => {
   }
 
   function handleChange(val: (MultiSelectItem | string)[]) {
-    const lastItem = val[val.length - 1];
+    const lastItem =
+      typeof val[val.length - 1] === 'string'
```

Comment:
please create a more specific name and you can use `lastIndex = ${specific name}.length - 1` to be cleaner.